config-templates/simple-file-binlog.yaml

```diff
@@ -0,0 +1,44 @@
+# # Global variables are applied to all deployments and used as the default value of
+# # the deployments if a specific deployment value is missing.
+global:
+  user: "tidb"
+  ssh_port: 22
+  deploy_dir: "/tidb-deploy"
+  data_dir: "/tidb-data"
+
+server_configs:
+  tidb:
+    binlog.enable: true
+    binlog.ignore-error: true
+
+pd_servers:
+  - host: 10.0.1.4
+  - host: 10.0.1.5
+  - host: 10.0.1.6
+tidb_servers:
+  - host: 10.0.1.1
+  - host: 10.0.1.2
+  - host: 10.0.1.3
+tikv_servers:
+  - host: 10.0.1.7
+  - host: 10.0.1.8
+  - host: 10.0.1.9
+
+pump_servers:
+  - host: 10.0.1.1
+  - host: 10.0.1.2
+  - host: 10.0.1.3
+drainer_servers:
+  - host: 10.0.1.12
+    config:
+      syncer.db-type: "file"
+      syncer.to.dir: "/path/to/save/binlog"
```

Comment:
worth to mention the `data-dir`
https://github.com/pingcap/tidb-binlog/blob/83984f4a49898f6120157e240ca6a666ddeeafde/cmd/drainer/drainer.toml#L150

Reply:
added this comment in https://github.com/pingcap/docs-cn/pull/5049/commits/638ccc0a3bf1358a96302f66a539161d0ab60a75